packages/ui/src/app/pages/WorkingGroups/UpcomingOpening.tsx

```diff
@@ -83,8 +84,8 @@ export const UpcomingOpening = () => {
                 from={opening.expectedStart}
               />
               <TokenValueStat
-                title={`Reward per ${rewardPeriod?.toString()} blocks`}
-                value={rewardPeriod?.mul(opening.rewardPerBlock)}
+                title={'Reward per week'}
+                value={rewardPeriod?.mul(opening.rewardPerBlock).mul(new BN(7))}
```

Comment:
The code now makes the assumption that `opening.rewardPerBlock` is a day , which is fine but in this case there's no need to use the api `rewardPerBlock` (because if the api returns something else than `14400` then `rewardPerBlock * 7` won't be a week anymore).

So IMO this would be a lot clearer:
```suggestion
                value={opening.rewardPerBlock.muln(A_WEEK / MILLISECONDS_PER_BLOCK}
```
Or even:
```suggestion
                value={opening.rewardPerBlock.muln(BLOCKS_PER_WEEK}
```
(With `BLOCKS_PER_WEEK` defined as a constant somewhere).